app/src/lumen/java/com/cliqz/browser/vpn/VpnPanel.java

```diff
@@ -54,11 +56,14 @@
     private View mVpnConnectButton;
     private TextView mVpnButtonTitle;
     private TextView mVpnButtonDesc;
-    private TextView mVpnStatusText;
     private ImageView mWorldMap;
     private Chronometer vpnTimer;
     private Handler mainHandler;
     private Boolean shouldAnimate = false;
+    private TextView featureOne;
+    private TextView featureTwo;
```

Comment:
please rename `featureOne` and `featureTwo` to something meaningful: are those messages? Can the user interact with them?